source/sync/local-to-sync.txt

```diff
@@ -94,33 +82,40 @@ language/platform:
 
 Copy Existing Data
 ------------------
-None of the data in your local {+realm+} will be automatically transferred to your 
-new synced {+realm+}; it is up to you to ensure the data you want to preserve is 
-copied to the new {+realm+}. How you go about doing this depends, in part, on your 
-specific needs, but the following diagram and steps outline the process for 
-copying all of the data from the local {+realm+} to the synced {+realm+}. If done 
-correctly, the impact on app performance after the initial copy will be 
-minimal.
+
+.. note:: Alternate SDK Implementations
+
+   Some SDKs support an easier implementation for adding sync to a
+   local-only realm than the steps outlined on this section.
+
+   - :ref:`Open Non-Synced Realm as a Synced Realm - Swift SDK <ios-open-non-synced-as-synced-realm>`
+   - :ref:`Open Existing Realm with Different Configuration - Node.js SDK <node-open-existing-realm-different-configuration>`
+   - :ref:`Open Existing Realm with Different Configuration - React Native SDK <react-native-open-existing-realm-different-configuration>`
+
+None of the data in your local realm automatically transfers to a
```

Comment:
Just a small comment: we characterize the the three SDK sections linked above as "adding sync to a local-only realm" but then in this paragraph we say "None of the data in your local realm automatically transfers." If folks use one of the three SDK options mentioned above, those actually do copy the data from the local realm to the synced realm. So we might want to reword this, or rework the line in the note above to provide info that "adding Sync to a local-only realm" also includes copying the data.